masschallenge/semantic-ui-theme
# AC 6573 Integrate MassChallenge semantic customizations into React/Front-end

**Changes introduced in [AC-6573](https://masschallenge.atlassian.net/browse/AC-6573)**
 - Added semantic docker files to the semantic repo


**How to test**
- ensure that you have pulled the changes in this branch, front-end and impact-api and are on branch AC-6573 in all these repos.
- run `make run-all-servers` to run both accelerate and front-end.
- make changes in the semantic-ui-theme repo and then changes will be seen in accelerate and in front-end

**For example**
- in semantic-ui-theme repo, navigate to the `message.overrides` file. 
  `src > themes > masschallenge > collections > message.overrides`
- insert the following property `* { background: #000}` and save.
- this will change the background color in accelerate and directory to black

**Note**
- Copied the semantic-ui docker files to this repo and making a followup ticket to remove them from Accelerate and have it point these ones in this repo.
